Jenkinsfile

```diff
@@ -2,223 +2,174 @@
+          env.DOCKER_TAG = dockerize_server.getDeployTag()
         }
       }
-    }
-    stage('Deploy') {
-      when { expression { stage_deploy } }
-      steps {
-        script {
-          // Checkout and run bundle install in the apps-marathon repo
-          deploy.checkoutAndInstall()
-
-          // deploy the service to the specified environment
-          deploy.deploy(deploy_service_pattern_server, deploy_environment, dockerize_server.getDeployTag())
-          deploy.deploy(deploy_service_pattern_secondary, deploy_environment, dockerize_secondary.getDeployTag())
+      post {
+        success {
+          script {
+            if (params.RELEASE_BUILD){
+              echo env.DOCKER_TAG // For now, just print the deploy tag in the console output -- later, communicate to release metadata service
+            }
+          }
         }
       }
     }
-
-    stage('Deploy PG Control Mirrors') {
+    stage('Dockerize Secondary') {
       when {
-        allOf {
-          expression { stage_deploy }
-          not { expression { return params.RELEASE_CUT} }
-        }
+        not { expression { isPr } }
       }
       steps {
         script {
-          deploy.checkoutAndInstall()
-
-          deploy.deploy("soql-server-mirror-control-pg1-staging", deploy_environment, dockerize_server.getDeployTag())
-          deploy.deploy("secondary-watcher-mirror-control-pg*", deploy_environment, dockerize_secondary.getDeployTag())
+          dockerize_secondary.docker_build(sbtbuild.getServiceVersion(), env.SERVICE_SHA, sbtbuild.getDockerPath(project_wd_secondary), sbtbuild.getDockerArtifact(project_wd_secondary))
+          env.SECONDARY_DOCKER_TAG = dockerize_secondary.getDeployTag()
         }
       }
+      post {
+        success {
+          script {
+            if (params.RELEASE_BUILD){
+              echo env.SECONDARY_DOCKER_TAG // For now, just print the deploy tag in the console output -- later, communicate to release metadata service
+            }
+          }
         }
-
-    stage('Deploy Citus Mirrors') {
+      }
+    }
+    stage('Deploys:') {
       when {
         allOf {
-          expression { stage_deploy }
-          not { expression { return params.RELEASE_CUT} }
+          not { expression { isPr } }
+          not { expression { return params.RELEASE_BUILD} }
         }
       }
-      steps {
-        script {
-          deploy.checkoutAndInstall()
-
-          deploy.deploy("soql-server-mirror-citus1-staging", deploy_environment, dockerize_server.getDeployTag())
-          deploy.deploy("secondary-watcher-mirror-citus*", deploy_environment, dockerize_secondary.getDeployTag())
+      stages {
+        stage('Deploy SoQL Server PG') {
+          steps {
+            script {
+              // uses env.DOCKER_TAG and deploys to staging by default
+              marathonDeploy(serviceName: env.DEPLOY_PATTERN, waitTime: '60')
```

Comment:
soql-server-pg* needs a high wait time for the deploy - https://jenkins-build.socrata.com/job/soql-postgres-adapter-cut/109/flowGraphTable/ - this is an example where it took 48 minutes to deploy soql-server-pg* to rc.